### PR TITLE
feat: add DELETE command to McKitchen

### LINKED ISSUE #11: Add delete command feature
Removes recipe from a text file in a user’s local computer

### PR DESCRIPTION
**Pull request by gigituang**
closes #11

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    private final Name toDelete;
+
+    /**
+     * Creates a DeleteCommand to delete the specified {@code Recipe}
+     * @param toDelete
+     */
+    public DeleteCommand(Name toDelete) {
+        this.toDelete = toDelete;
+    }
+
```
Comment (yusufaine) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:39]: don't forget to add the javadocs for getRecipeIndex()

**Review by yusufaine — APPROVED** [review]
Looks good to me, I left a comment regarding the JavaDocs in `DeleteCommand`, but that aside, LGTM. 🌮

**Review by blaketingyu — APPROVED** [review]
LGTM! 🌮 Good job on the delete command! 👍

**Review by Oddcorner — APPROVED** [review]
LGTM! 🌮

**Pull request merged**